release-team/role-handbooks/patch-release-manager/README.md

```diff
@@ -146,6 +207,7 @@ For each cherry-pick request:
       For cherry-picks that are clearly justified and low risk in your judgment,
       you can directly apply the `approved` label as long as the parent PR was
```

Comment:
I'm just curious if we want to encourage this behaviour of adding the `approved` label manually? I don't have strong opinions on this but I didn't realize the patch manager had those powers. Wouldn't it be safer to require explicit lgtm + approval in all cases? :thinking:

Reply:
People in the release manager team have write/admin accesses to some repos. I think it is reasonable to do this, with cautions of course. Otherwise, patch manager have to wait for someone in the owners file to approve in every cherry-pick PR.

Reply:
To be clear, I think we can have more details about _low risk_ here if needed, but adding the label manually may be nessary sometimes. Also, after read more around these paragraphs, I found the process described here was not quite the same as I observed. What I thought was the reviewer and the approver should be those in the original PR or someone from the SIG in most cases. 🤔

Reply:
> Also, after read more around these paragraphs, I found the process described here was not quite the same as I observed. What I thought was the reviewer and the approver should be those in the original PR or someone from the SIG in most cases. 

+1, the way it is written now suggests that the patch release manager can lgtm + approve + cherry-pick approve as well....

```
If the parent PR has merged, comment /lgtm to apply the lgtm label and notify the author that you've reviewed the cherry-pick request.

For cherry-picks that are clearly justified and low risk in your judgment, you can directly apply the approved label as long as the parent PR was approved and merged into master.
```

Reply:
Given what I read from the current playbook, they can. It is consistent here. But the playbook also said patch release managers can always work with other SIGs :) This explains what we’ve seen before when patch release managers asked SIGs to approve cherry-picks.

In SIG-Release, we’ve agreed we don’t want to put a branch on one person’s shoulders for nine months, so we have two since 1.13 and will have a team in 1.14 :)

Reply:
To me, adding `approve` makes sense only as an exception, not as the normal mode of operation (if it's really obvious and urgent, you may; otherwise, don't). In particular, I'd like to avoid any expectation that PRM will be doing that. Just because a PR was approved on master doesn't necessarily mean code owners agree it should be cherry-picked, or even that it's safe to cherry-pick.

LGTM seems much less controversial, as it can be applied by a larger set of people anyway, but I agree the practice is different. Not sure if we should change the practice or the docs...

Reply:
@aleksandra-malinowska Thanks for your input here! We may wait for other reviews with their practices to decide on this.

Reply:
FWIW in this PR I've tried to stick to updating the doc to reflect the current (often ambiguous still) process that is performed.  We DEFINITELY intend to follow with more conversation around improving and disambiguating the process.